src/main.lua

```diff
@@ -70,7 +70,8 @@ local ActionMap = {
 	VaultedTransfer = "Vaulted-Transfer",
 	ExtendVault = "Extend-Vault",
 	IncreaseVault = "Increase-Vault",
-	BuyRecord = "Buy-Record",
+	BuyRecord = "Buy-Record", -- TODO: standardize these as `Buy-Name` or `Upgrade-Record`
+	UpgradeName = "Upgrade-Name", -- TODO: may be more aligned to `Upgrade-Record`
```

Comment:
Honestly, upgrade is a bit ambiguous. `Permabuy-Record` seems more fitting, but not sure if that creates confusion for users as to whether they should use Buy-Record or Permabuy-Record for an initial (non-lease or "upgrade") purchase.